assets/js/app.js

```diff
@@ -0,0 +1,53 @@
+import Book from '../modules/book.js';
+
+import {
+  navlist, navadd, navcon, listfun, addfun, confun,
+} from '../modules/nav.js';
+
+// eslint-disable-next-line no-unused-vars
+// const displayTime = () => {
+//   document.getElementById('current-date').innerText = new Date().toLocaleString();
+//   setTimeout(displayTime, 1000);
+// };
+// displayTime();
+
```

Comment:
- [ ] Let's remove the commented code, it's part of making our codebase look clean best practice.

Reply:
Thanks for reviewing my project @elisha2kyakpo1 :pray: 
I'll remove these comments.